packages/components/src/components/segmented-control/segmented-control/segmented-control-utils.ts

```diff
@@ -17,6 +17,9 @@ export type SegmentedControlBackgroundColor = (typeof SEGMENTED_CONTROL_BACKGROU
 
 export type SegmentedControlUpdateEvent = { value: string | number };
 
+export const SEGMENTED_CONTROL_COLUMNS = [1, 2, 3, 4, 5, 'auto'];
+export type SegmentedControlColumns = (typeof SEGMENTED_CONTROL_COLUMNS)[number];
```

Comment:
When using this type you always type a single one, so the type shouldn't be plural.
```suggestion
export type SegmentedControlColumn = (typeof SEGMENTED_CONTROL_COLUMNS)[number];
```